app/components/header.tsx

```diff
@@ -91,15 +82,13 @@ export default function Header() {
           </MenuList>
         </Menu>
       </Show>
-      <Show below="lg">
-        <Link to={{ pathname: '/' }}>
-          <Flex justifySelf="flex-start">
-            <LockIcon color="gray.600" />
-          </Flex>
-        </Link>
-      </Show>
+      <Show below="lg"></Show>
+
+      <Heading as="h1" size="xl" color="white" width="100%" textAlign="center">
+        My.Custom.Domain
+      </Heading>
```

Comment:
I think you can now link this to `/`

Reply:
oh yeah I have. It also has a functionality that I can assign it `as={Link}` and `to={{pathname: '/'}}`, but  I couldn't make it so the test doesn't fail

Reply:
I tried running this but it does not link to anything

Reply:
oh you caught into it, I forgot to add the the Link component